convert2rhel/checks.py

```diff
@@ -75,77 +89,79 @@ def perform_pre_ponr_checks():
 
 
 def check_convert2rhel_latest():
-    """Make sure that we are running the latest version of convert2rhel"""
-    gpg_path = os.path.join(utils.DATA_DIR, "gpg-keys", "RPM-GPG-KEY-redhat-release")
-    ssl_cert_path = os.path.join(utils.DATA_DIR, "redhat-uep.pem")
-    repo_content = (
-        "[convert2rhel]\n"
-        "name=Convert2RHEL Repository\n"
-        "baseurl=https://cdn.redhat.com/content/public/convert2rhel/$releasever/x86_64/os/\n"
-        "gpgcheck=1\n"
-        "enabled=1\n"
-        "sslcacert=%s\n"
-        "gpgkey=file://%s\n" % (ssl_cert_path, gpg_path)
-    )
+    """Make sure that we are running the latest downstream version of convert2rhel"""
+    logger.task("Prepare: Checking if this is the latest version of convert2rhel")
+
+    if not system_info.has_internet_accsess:
+        logger.warning("Skipping the check because no internet connection has been detected.")
+        return
```

Comment:
```suggestion
    if not system_info.has_internet_access:
        logger.warning("Skipping the check because no internet connection has been detected.")
        return
```

Reply:
thank you for pointing this out, I was actually working the unit test and noted changed that!